data_diff/database.py

```diff
@@ -381,7 +381,13 @@ def to_string(self, s: str):
 
     def _query(self, sql_code: str) -> list:
         "Uses the standard SQL cursor interface"
-        return _query_conn(self._conn, sql_code)
+        c = self._conn.cursor()
```

Comment:
If this is Presto-specific, it should only be used for Presto

Reply:
I'm not sure I understand what you mean 👀  This is on the Presto driver

Reply:
Okay, my bad.